app/app/search.py

```diff
@@ -154,38 +161,40 @@ def find_in_predicted(
 
 @cache
 def get_chemical_classes() -> list[Tuple[str, str]]:
-    all_compounds = select(ValidatedCompounds.compound_id).distinct().subquery()
+    all_compounds = select(ValidatedCompounds.compound_id).distinct()
     stmt = select(Compounds.chemical_class).filter(Compounds.compound_id.in_(all_compounds)).distinct()
```

Comment:
Sorting helps the user find the options:
```suggestion
    stmt = select(Compounds.chemical_class).filter(Compounds.compound_id.in_(all_compounds)).distinct().order_by(collate(Compounds.chemical_class, 'NOCASE'))
```

Reply:
This should be resolved now